rest/data/test_results.go

```diff
@@ -235,11 +258,27 @@ func getAPITestResultFromBucket(ctx context.Context, bucket pail.Bucket, testNam
 	return apiResult, ctx.Err()
 }
 
-func convertToDBOpts(opts TestResultsOptions) dbModel.TestResultsFindOptions {
+func getTestResultsBucket(ctx context.Context, path, prefix string) (pail.Bucket, error) {
```

Comment:
Can this be called `getMockTestResultsBucket` so nobody tries to use it in the real DB connector?

Reply:
Made it a function on the `MockConnector`.